lnwallet/channel.go

```diff
@@ -159,13 +159,9 @@ func (e *ErrCommitSyncLocalDataLoss) Error() string {
 type channelState uint8
 
 const (
-	// channelPending indicates this channel is still going through the
-	// funding workflow, and isn't yet open.
-	channelPending channelState = iota // nolint: unused
-
 	// channelOpen represents an open, active channel capable of
 	// sending/receiving HTLCs.
-	channelOpen
+	channelOpen channelState = iota
```

Comment:
this is gonna cause the `channelOpen` value to be changed?

Reply:
Yeah, but since it is never persisted that's OK. Perhaps to lower the burden even further I can make it so that we don't change the value itself.